standard_forms/outbound-invoice-portrait-with-logo/latest/src/header.html

```diff
@@ -1,49 +1,75 @@
-{% macro dateOrNA(datetime) %}
-{% if datetime %}{{ datetime | date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else %}{{t(k="messages.not-applicable", f="N/A")}}{% endif %}
-{% endmacro input %}
+{% macro dateOrNA(datetime) %} {% if datetime %}{{ datetime |
+date(format="%d/%m/%Y", timezone=arguments.timezone) }}{% else
+%}{{t(k="messages.not-applicable", f="N/A")}}{% endif %} {% endmacro input %}
 
-<table class="header_image_section" style="width: 100%;height: 98%;">
-    <tr>
-        <th style="text-align:left;width:10%;height:70%;">
-            <img class="logo" src="{{ data.store.logo }}" />
-        </th>
-        <th style="text-align:left;width:20%;">
-            <span style="font-weight:bold;">{{ data.store.storeName }}</span><br>
-            <span style="font-size:6pt;">{{ data.store.name.address1 }}</span><br>
-            <span style="font-size:6pt;">{{ data.store.name.address2 }}</span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.phone",f="Telephone")}}: {{ data.store.name.phone }}
-            </span><br>
-            <span style="font-size:6pt;">
-                {{t(k="label.email",f="Email")}}: {{ data.store.name.email }}
-            </span>
-        </th>
-        <th style="text-align:right;width:70%;font-size:28pt;">
-            {{ t(k="label.outbound-shipment", f="Outbound Shipment") }}<br>
-        </th>
-    </tr>
+<!-- Main form has 7 columns, so we'll work with that here (means excel report header and 
+ data table are the same width) -->
+<table class="header_section">
+  <tr class="header_image_section">
+    <td rowspan="5">
+      <img class="logo" src="{{ data.store.logo }}" />
+    </td>
+    <td colspan="2" class="bold">{{ data.store.storeName }}</td>
+    <td></td>
+    <td colspan="3" rowspan="2" class="title">
+      {{ t(k="label.outbound-shipment", f="Outbound Shipment") }}
+    </td>
+  </tr>
+  <tr>
+    <td>{{ data.store.name.address1 }}</td>
+  </tr>
+  <tr>
+    <td>{{ data.store.name.address2 }}</td>
+  </tr>
+  <tr>
+    <td>{{t(k="label.phone",f="Telephone")}}:</td>
+    <td>{{ data.store.name.phone }}</td>
+  </tr>
+  <tr>
+    <td>{{t(k="label.email",f="Email")}}:</td>
+    <td>{{ data.store.name.email }}</td>
+  </tr>
+  <tr></tr>
+</table>
+<table class="header_section">
+  <tr>
+    <td>{{t(k="label.customer", f="Customer")}}:</td>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
```

Comment:
this is probs the main bit of jank - if you want stuff on the far side of the page, need to add these in-between cells, where we used to just do space-between 👀 

Could use a merged cell here i.e. one `<td colspan="4"></td>`